### DISCUSSION TITLE
Premium/Enterprise support for self-hosted system

### DISCUSSION THREAD

**Discussion by BusDar** [chatwoot/chatwoot#8318] [General]
Hello Chatwoot team @pranavrajs @nithindavid @sojan-official ,

I’m trying to reach Mr.Pranav (through sales@chatwoot.com) regarding premium/enterprise support for self-hosted system, but I have not gotten a response for almost two weeks.
Who can help to figure out this issue?

Regards,
Darius

**Comment by pranavrajs** [comment]
@BusDar, I couldn't find any missing emails at the moment. Can you please forward it to pranav [at] chatwoot.com? I will take care of it.

**Reply by DariusBus** [reply]
Hi @pranavrajs , 

I have forwarded our email conversation to your personal email, but you should check sales email as there should be historical records of our conversation from darius.busilas [at] teamlt.lt.
